extensions/arc/src/localizedConstants.ts

```diff
@@ -334,4 +334,23 @@ export const userCancelledError = localize('arc.userCancelledError', "User cance
 export const invalidConfigPath = localize('arc.invalidConfigPath', "Invalid config path");
 export const loadingClusterContextsError = (error: any): string => localize('arc.loadingClusterContextsError', "Error loading cluster contexts. {0}", getErrorMessage(error));
+
+// Upgrade
+export const upgradeManagement = localize('arc.upgradeManagement', "Upgrade Management");
+export const currentControllerVersion = localize('arc.currentControllerVersion', "Current controller version: ");
+export const availableUpgrades = localize('arc.availableUpgrades', "Available Upgrades");
+export const availableUpgradesDescription = localize('arc.availableUpgradesDescription', "Available upgrades for this resource are listed below. You can apply upgrades by selecting the upgrade button. Learn more about each of these releases at the");
+export const versionLog = localize('arc.versionLog', "version log.");
+export const onlyNextImmediateVersion = localize('arc.onlyNextImmediateVersion', "Currently, only upgrading to the next immediate version is supported.");
+export const version = localize('arc.version', "Version");
+export const releaseDate = localize('arc.releaseDate', "Release Date");
+export const releaseNotes = localize('arc.releaseNotes', "Release Notes");
+export const upgrade = localize('arc.upgrade', "Upgrade");
+export const upgradeDataController = localize('arc.upgradeDataController', "Upgrade Data Controller");
+export const areYouSure = localize('arc.areYouSure', "Are you sure you want to apply the selected upgrade?");
+export const upgradeDialog = localize('arc.upgradeDialog', "During a data controller upgrade, portions of the data control plane such as Custom Resource Definitions (CRDs) and containers may be upgraded. An upgrade of the data controller will not cause downtime for the data services (SQL Managed Instance or PostgreSQL server).");
+export const monitorUpgrade = localize('arc.monitorUpgrade', "You can check the status of the upgrade by running the following command:");
+export const monitorUpgradeCommand = localize('arc.monitorUpgradeCommand', "kubectl get datacontrollers -A");
 export function errorListingLogAnalyticsWorkspaces(error: any): string { return localize('arc.errorListingLogAnalyticsWorkspaces', "Error listing Log Analytics workspaces {0}", getErrorMessage(error, true)); }
+export const noUpgrades = localize('arc.noUpgrades', 'The current version is the latest version. No upgrades available.');
+export const upgrading = localize('arc.upgrading', "Data controller is being upgraded. You can check the status of the upgrade by running the following command: 'kubectl get datacontrollers -A'");
```

Comment:
Please add a comment the localize function indicating that "kubectl get datacontrollers -A" shouldn't be localized